CONTRIBUTING.md

```diff
@@ -70,7 +77,7 @@ Reviewers will run through roughly the following checklist:
 - Are boundary cases considered?
 - Are error cases handled?
 
-(Once we have some tests, the reviewer will check that all the tests are passing,
+(the reviewer will check that all the tests are passing,
```

Comment:
maybe just pull this into the list of bullet points above. speaking of which, does that list of reviewer things to look for actually track with how reviews are done? if not, should we modify the list or just remember to use the list when reviewing? cc @lmorchard